gobblin-core/src/main/java/org/apache/gobblin/writer/FsDataWriter.java

```diff
@@ -262,7 +262,7 @@ public void commit()
     LOG.info(String.format("Moving data from %s to %s", this.stagingFile, this.outputFile));
     // For the same reason as deleting the staging file if it already exists, overwrite
     // the output file if it already exists to prevent task retry from being blocked.
-    HadoopUtils.renamePath(this.fileContext, this.stagingFile, this.outputFile, true);
+    HadoopUtils.renamePath(this.fs, this.stagingFile, this.outputFile, true);
```

Comment:
Did you forget to remove the fileContext member variable from this class?

Reply:
This is intentional, as we don't know if there is any subclass use this member. So I leave it to make backward consistency